generators/workspace/index.js

```diff
@@ -462,15 +462,17 @@ class Generator extends Base {
   }
 
   end() {
-    this.log('\n\nuseful commands: ');
-    this.log(chalk.red(' docker-compose up'), '        ... starts the system');
-    this.log(chalk.red(' docker-compose restart'), '   ... restart');
-    this.log(chalk.red(' docker-compose stop'), '      ... stop');
-    this.log(chalk.red(' docker-compose build api'), ' ... rebuild api (in case of new dependencies)');
-
-    this.log('\n\nnext steps: ');
-    this.log(chalk.red(' npm install'));
-    this.log(chalk.red(' docker-compose up'));
+    if (!this.options.noLogging) {
```

Comment:
- [ ] A better name would probably be:
   ```suggestion
       if (!this.options.skipNextStepsLog) {
   ```
- [ ] Also register the option in the constructor similar to `wsName` (with default value and description).